x-pack/metricbeat/module/iis/application_pool/_meta/fields.yml

```diff
@@ -54,19 +54,90 @@
       description: >
         Common Language Runtime overview.
       fields:
-        - name: finallys_per_sec
-          type: float
+        - name: exceptions
```

Comment:
@stefans-elastic - We cannot make the existing fields for exceptions as group because this will be a breaking change for existing users. As mentioned in my previous comment this is the reason I think either we can make the memory and lockandthreads as group fields by category or not add group by category for consistency.

Reply:
sorry, I must have missed you mentioning this in the previous comment
I've corrected it